### PR TITLE
Adds correct version number to automated install flow

### PR DESCRIPTION
**Pull request by ctauchen**

### CODE REVIEW

**Comment by ctauchen** [comment]
CC @gcosgrave @tmjd

File: calico-cloud/get-started/connect/install-automated.mdx [hunk 1]
```diff
@@ -96,7 +96,7 @@ These commands can be added to any automated workflow.
     # clusterName is the unique name this cluster will have in Calico Cloud
     clusterName: my-cluster
     # calicoCloudVersion is the version to install
-    calicoCloudVersion: v19.1.0
+    calicoCloudVersion: {{cloudUserVersion}}
```
Comment (tmjd) [inline calico-cloud/get-started/connect/install-automated.mdx:99]: Are you purposefully not adding this to the versioned version of this file?

Comment (ctauchen) [reply calico-cloud/get-started/connect/install-automated.mdx:99]: Nope, managed to missing pushing that change. Thanks for spotting this! Change added.

**Comment by gcosgrave** [comment]
LGTM

**Pull request merged**